MantidQt/CustomInterfaces/test/EnggDiffractionViewMock.h

```diff
@@ -53,11 +53,16 @@ class MockEnggDiffractionView
   // std::string currentPlotType
   MOCK_CONST_METHOD0(currentPlotType, int());
 
-  // virtual std::string newVanadiumNo() const;
-  MOCK_CONST_METHOD0(newVanadiumNo, std::string());
+  // virtual std::vector<std::string> newVanadiumNo() const;
+  MOCK_CONST_METHOD0(newVanadiumNo, std::vector<std::string>());
 
-  // virtual std::string newCeriaNo() const;
-  MOCK_CONST_METHOD0(newCeriaNo, std::string());
+  // virtual std::vector<std::string> newCeriaNo() const;
+  MOCK_CONST_METHOD0(newCeriaNo, std::vector<std::string>());
+
+  // virtual std::vector<std::string> qListToVector(QStringList list, bool
+  // validator) const;
+  MOCK_CONST_METHOD2(qListToVector, std::vector<std::string>(QStringList list,
+                                                             bool validator));
```

Comment:
I think that compilation is failing because QStringList is not included here? That type is probably not know in this file, as it only includes a couple of small headers.